### DISCUSSION TITLE
Subscriber Signup

### DISCUSSION THREAD

**Discussion by justynpride** [mettle/sendportal#120] [Help] [answered]
Hi. I'm new to this and a basic coder. I've successfully installed the standalone app and it is fully working. My question is how can I get subscribers to signup through my website. I can see how I can manually add and import but not clear on an embedded form. I'm sure it is more API related but that is not my currently skill base, though happy to learn from a helpful guide!

**Comment by JonoB** [comment, marked as answer]
We don't have an embedded form functionality at the moment, so Subscribers can only be added (i) manually, (ii) import or (iii) via the API.

We've got embedded forms on the backlog though!

**Reply by justynpride** [reply]
Thanks for the helpful info. I will probably need to wait to use the system until the embedded forms in there. My skill isn't currently API friendly.
